src/plugins/kibana_usage_collection/server/plugin.ts

```diff
@@ -82,11 +85,13 @@ export class KibanaUsageCollectionPlugin implements Plugin {
     this.uiSettingsClient = uiSettings.asScopedToClient(savedObjectsClient);
     core.metrics.getOpsMetrics$().subscribe(this.metric$);
     this.coreUsageData = core.coreUsageData;
+    startTrackingEventLoopDelaysUsage(this.savedObjectsClient, this.stopMonitoringEventLoop$);
```

Comment:
nit: `this.stopMonitoringEventLoop$.asObservable()` to ensure no-one triggers it within `startTrackingEventLoopDelaysUsage` function

Reply:
Interesting. I didnt know this